tests/test_touringcar/test_view.py

```diff
@@ -0,0 +1,56 @@
+import geojson
+import pytest
+from django.core.exceptions import ValidationError
+from model_bakery import baker
+
+from touringcar.model import Bericht
+from touringcar.serializer import BerichtFilterSerializer, BerichtSerializer
+
+
+@pytest.fixture
+def bericht():
+    return baker.prepare(Bericht, is_live=True, startdate="2023-11-15", enddate = "2023-11-30")
+
+@pytest.fixture
+def bericht_error():
+    return baker.prepare(Bericht, is_live=True, startdate="2023-11-15", enddate = "2023-11-04")
+
+
+@pytest.mark.django_db
+def test_error_start_enddate(bericht_error):
+    with pytest.raises(ValidationError) as e:
+        bericht_error.save()
+    assert "{'enddate': ['enddate can not be before startdate.']}" in str(e.value)
```

Comment:
Dit voelt wat gek. Ik zou die `e.value` met `json.loads()` en dan de value van `enddate` asserten.

Reply:
print(e) geeft <ExceptionInfo ValidationError({'enddate': ['enddate can not be before startdate.']}) tblen=3> 
de message ophalen lukt mij alleen via e.value...